README.md

```diff
@@ -71,6 +71,10 @@ This driver supports Kerberos authentication by setting up the Kerberos fields i
 
 Please refer to the [Coordinator Kerberos Authentication](https://trino.io/docs/current/security/server.html) for server-side configuration.
 
+#### Bearer authentication
+
+This driver supports Bearer authentication (also called token authentication) by setting it in the [Config](https://godoc.org/github.com/trinodb/trino-go-client/trino#Config) struct.
```

Comment:
This needs an example, especially if users are supposed to prefix any tokens with `Bearer`.